src/convert.ts

```diff
@@ -194,18 +194,17 @@ function crdaToResult(
 
 function fetchResults(
     publiclyAvailableVulnerabilities: CrdaPubliclyAvailableVulnerability[],
-    manifestFile: string, startLine: number, isTransitive: boolean,
+    manifestFile: string, startLine: number, isDirect: boolean,
     dependencyName: string, dependencyVersion: string, recommendedVersion: string,
     latestVersion: string,
 ): [ sarif.Result[], string[] ] {
     const results: sarif.Result[] = [];
     const ruleIds: string[] = [];
     publiclyAvailableVulnerabilities.forEach((publiclyAvailableVulnerability) => {
         const ruleId = publiclyAvailableVulnerability.id;
-        let textMessage = `This file introduces a vulnerability ${publiclyAvailableVulnerability.title} with `
+        let textMessage = `This file introduces a ${publiclyAvailableVulnerability.title} vulnerability with `
             + `${publiclyAvailableVulnerability.severity} severity.\n`
-            + `Vulnerability present at ${dependencyName}\n`
-            + `Version: ${dependencyVersion}\n`;
+            + `Vulnerable ${isDirect ? "" : "transitive"} dependency is ${dependencyName} ${dependencyVersion}.\n`;
```

Comment:
if it is direct, it will say `Vulnerable  dependency` with two spaces

Reply:
I think it makes more sense to put the newlines here instead
![Uploading image.png…]()

Reply:
You mean at the start of the string

Reply:
like `\n${textMessage}Recommended Version: ${recommendedVersion}`?

Reply:
Yes so that when it's only the first section, there isn't an extra newline at the end.

Reply:
But it would be

`${textMessage}\nNew text goes here`

Reply:
yeah